shared/templates/file_permissions/oval.template

```diff
@@ -32,6 +34,9 @@
         <unix:filename operation="pattern match">{{{ FILE_REGEX }}}</unix:filename>
       {{%- else %}}
         <unix:filename xsi:nil="true" />
+      {{%- if ALLOW_STRICTER_PERMISSIONS %}}
+        <filter action="include">state_file_permissions{{{ FILEID }}}_mode_not_{{{ FILEMODE }}}</filter>
+      {{%- endif %}}
```

Comment:
The use of `filter action=include` here is not wrong, but may require some explanation as to why we go the approach of "defining that the object with a certain state doesn't exist" opposed "defining the object and checking that it doesn't match a state".

Reply:
The filter should be kept until 1.3.5 is widely deployed, but there is no harm in keeping the filter indefinitely.

Reply:
I'll add a comment. I'll also try to unify this filter with the existing one that is being used for the file option (when the check is configured to look for a file)